backend/src/apiserver/main.go

```diff
@@ -98,13 +99,13 @@ func main() {
 	)
 	err = loadSamples(resourceManager)
 	if err != nil {
-		glog.Fatalf("Failed to load samples. Err: %v", err)
+		glog.Fatalf("Failed to load samples.This can be safely ignored if sample configurations are not needed. Err: %v", err)
```

Comment:
Isn't the Fatalf function here going to exit either way with `os.Exit(2)` so basically it can't be ignored? Same question applies for line 108.